website/content/v1.6/kubernetes-guides/configuration/local-storage.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Local Storage"
+description: "Using local storage for Kubernetes workloads."
+---
+
+Using local storage for Kubernetes workloads implies that the pod will be bound to the node where the local storage is available.
+Local storage is not replicated, so in case of a machine failure contents of the local storage will be lost.
+
+> Note: when using `EPHEMERAL` Talos partition (`/var`), make sure to use `--preserve` set while performing upgrades, otherwise you risk losing data.
+
+## `hostPath` mounts
+
+The simplest way to use local storage is to use `hostPath` mounts.
+When using `hostPath` mounts, make sure the root directory of the mount is mounted into the `kubelet` container:
```

Comment:
Great long-overdue documentation ❤️

Reply:
yeah, I feel people are confused and think the minimum is to use OpenEBS